iceoryx_binding_c/include/iceoryx_binding_c/sleep_for.h

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
```

Comment:
why do we need this sleep? Is this our business?

Reply:
moved into example where its needed